std_execution.bs

```diff
@@ -3643,16 +3698,16 @@ namespace std::execution {
            class E = no_env,
            template &lt;class...> class Tuple = <i>decayed-tuple</i>,
            template &lt;class...> class Variant = <i>variant-or-empty</i>>
-      requires typed_sender&lt;S, E>
+      requires sender&lt;S, E>
     using value_types_of_t =
-      typename sender_traits_t&lt;S, E>::template value_types&lt;Tuple, Variant>;
+      typename completion_signatures_of_t&lt;S, E>::template value_types&lt;Tuple, Variant>;
```

Comment:
I was thinking that we could make the value_types&error_types&sends_stopped unspecified implementation details - even if we kept the sender_value_types_t etc.. alias'.

As I continued it seemed that there was still a lot of the spec directly dependent on the value_types&error_types&sends_stopped instead of through the alias'.

I think it is a nice-to-have to make the internals impl defined.

Reply:
I'm going to leave it for now. Bryce needs an R4 for the polls ASAP.